(trunk) PXC-4394 Misleading SST version comparison message

aybek opened the pull request:
https://perconadev.atlassian.net/browse/PXC-4394

Added a validation to ensure the pxb version could be retrieved before performing the version comparison.

kamil-holubicki commented:
There is not PR for 8.4 or do I miss something?

Is it possible to have 8.0 and 8.4 PRs as GCA? Probably for trunk it is not possible, so cherry pick is enough (or maybe possible?)

aybek committed:
PXC-4394 Misleading SST version comparison message
https://perconadev.atlassian.net/browse/PXC-4394

Added a validation to ensure the pxb version could be retrieved before performing the version comparison.

aybek committed:
Merge branch 'pxc-4394-8.0' into pxc-4394-8.4

aybek committed:
Merge branch 'pxc-4394-8.4' into pxc-4394-trunk

aybek commented:
https://pxc.cd.percona.com/view/8.0%20parallel%20MTR/job/pxc-8.x-pipeline-parallel-mtr/491/testReport/
